### ISSUE TITLE
Graph clipping in UWP

### ISSUE THREAD

**Issue by mjeanrichard** [oxyplot/oxyplot#1079]
### Steps to reproduce

1. Create a new UWP Project
2. Paste this into code behind:
```csharp
    public sealed partial class MainPage : Page
    {
        private byte[] DataPoints = new byte[] {0, 5, 100, 105, 102, 10, 12, 13, 180, 181, 90, 0};

        public MainPage()
        {
            PlotModel pm = new PlotModel();
            LineSeries lineSeries = new LineSeries();
            pm.Series.Add(lineSeries);
            pm.Axes.Add(new LinearAxis { Maximum = 260, Minimum = 0, Position = AxisPosition.Left, Title = "Intensität" });
            pm.Axes.Add(new LinearAxis { Position = AxisPosition.Bottom, Title = "Dauer [ms]" });

            lineSeries.Points.AddRange(DataPoints.Select((b, i) => new DataPoint(i, b)));
            Data = pm;

            this.InitializeComponent();
        }

        public PlotModel Data { get; set; }
    }

```
3. Put this in the XAML
```xaml
<Page
    x:Class="OxyRepro.MainPage"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:OxyRepro"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    xmlns:windows="using:OxyPlot.Windows"
    mc:Ignorable="d">

    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
        <windows:PlotView Model="{x:Bind Data}" />
    </Grid>
</Page>
```
4. Run

See also attached Repro. ([OxyRepro.zip](https://github.com/oxyplot/oxyplot/files/910808/OxyRepro.zip))

Platform:
.NET version: UWP (Windows 10)

### Expected behaviour

- Complete Graph is visible.
- Zoom works correctly

### Actual behaviour

- See marked areas in Screenshot (graph Line is missing)
- When zooming part of the graph disappears

![oxyrepro](https://cloud.githubusercontent.com/assets/778448/24872585/3619bdbe-1e1e-11e7-91fe-baf87d4cfea3.PNG)

**Comment by Nicothekrug** [comment]
This realy seems to be related to Creator's Update, since i have an application using Oxyplot, working fine on standard windows for 20+ PCs/tablets.

In parallel 1 PC 1 tablet updated with Creator's Update, the two have this weird plot cropping. Might be a XAML framework-related bug

**Comment by mjeanrichard** [comment]
I did some debugging, but since I do not really know the code in detail I can only make some observations:

If I reset the Clipping by adding a `rc.ResetClip()` in `RenderingExtensions.DrawClippedLine` on line 100 the lines are rendered fine. Even zoom and pan work as expexted:
```csharp
... 
if (rc.SetClip(clippingRectangle))
{
     rc.ResetClip();
     rc.DrawLine(points, stroke, strokeThickness, dashArray, lineJoin, aliased);
...
```

Even without the above hack the clipping does not prevent the lines from drawing over the axis when the plot is panned. Is this expected? 
![image](https://cloud.githubusercontent.com/assets/778448/25780007/2ce79ef6-3321-11e7-95c9-c1d13a469d30.png)

**Comment by csabar** [comment]
I think this is related: https://social.msdn.microsoft.com/Forums/en-US/d24e2dc7-78cf-4eed-abfc-ee4d789ba964/windows-10-creators-update-uielement-clipping-issue?forum=wpdevelop

**Comment by mjeanrichard** [comment]
Yes, I agree.  It is now confirmed from MS. The workaround described there does work indeed. (Add a ``.CompositeMode = ElementCompositeMode.SourceOver;`` to UIElements created in the RenderContext).

**Comment by mjeanrichard** [comment]
If anyone else need the packages with the workaround, they are available here:
https://ci.appveyor.com/project/MJeanRichard/oxyplot/build/artifacts

**Comment by GeertvanHorrik** [comment]
I have something similar:

![oxyplot](https://user-images.githubusercontent.com/1246444/26976101-c181e86c-4d22-11e7-8a2d-8440fa95b07a.gif)

**Comment by GeertvanHorrik** [comment]
@mjeanrichard have you created a PR with a fix for OxyPlot?

**Comment by GeertvanHorrik** [comment]
Somehow your repository is not in this list:

https://github.com/oxyplot/oxyplot/compare

**Comment by GeertvanHorrik** [comment]
I've been having users complain about this issue since the fast ring creators update (so for a *long* time). We could do a Platform version check, but I suggest we use this hotfix until this gets fixed for real (the delta between saying and doing can be quite long).

I'd rather have "bad performance" but working graphs than crappy graphs that are superfast.
